runtime/core/exec_aten/util/tensor_util.h

```diff
@@ -484,6 +486,30 @@ inline bool tensor_is_type(
   return true;
 }
 
+inline bool tensor_is_type(
+    executorch::aten::Tensor t,
+    const std::vector<executorch::aten::ScalarType>& dtypes) {
```

Comment:
The runtime must not use streams, vectors or any other type that allocates memory dynamically: see https://pytorch.org/executorch/main/portable-cpp-programming.html for some commentary on this.

In this case you may be able to use a `Span<ScalarType>` since the container size is known at construction time.

Reply:
noted, I will use the Span container